docs/static/files/fetch/examples/relationships/influxdb

```diff
@@ -1,11 +1,11 @@
 {
-    "service": "influxdb18",
+    "service": "influxdb23",
```

Comment:
For now, I think we shouldn't make this change. 

This file is pulled from the https://examples.docs.platform.sh project. Going forward, I think it makes sense to dismantle that project and have everything in docs - but for now we should keep the behavior the same everywhere.

Reply:
Plus, it gets overwritten during the deployment anyway https://bffpr-3086-5xcwfqi-652soceglkw4u.eu-3.platformsh.site/add-services/influxdb.html#relationship-reference